Tools/CISupport/ews-build/steps.py

```diff
@@ -1754,17 +1761,36 @@ def start(self):
             committer = self.getProperty('patch_committer', '').lower()
 
         if not self.is_committer(committer):
-            self.fail_build(committer, 'committer')
+            self.fail_invalid_status(committer, 'committer')
             return None
         self._addToLog('stdio', f'{committer} is a valid commiter.\n')
 
         if pr_number:
             reviewers = self.get_reviewers(pr_number, self.getProperty('repository', ''))
-            if any([self.is_reviewer(reviewer) for reviewer in reviewers]):
-                reviewers = list(filter(self.is_reviewer, reviewers))
         else:
             reviewer = self.getProperty('reviewer', '').lower()
             reviewers = [reviewer] if reviewer else []
+
+        remote = self.getProperty('remote', DEFAULT_REMOTE)
+        validators = self.VALIDATORS_FOR.get(remote, [])
+        if validators and not any([validator in reviewers for validator in validators]):
+            user_prefix = "@" if pr_number else ""
+            if len(validators) == 1:
+                validator_list = f'{user_prefix}{validators[0]}'
+            else:
+                validator_list = f'{", ".join(f"{user_prefix}{v}" for v in validators[:-1])} or {user_prefix}{validators[-1]}'
+            reason = f"Landing changes on '{remote}' requires validation from {validator_list}"
```

Comment:
Nit: `Landing changes on apple requires validation` might be a confusing error message for users. Might want to make it little more explicit, atleast say on `apple remote` instead of `apple`.

Also, adding a new term `validation`/`validator` might be confusing. How does someone validates? Isn't it just approval from specific users. Maybe the error message should just say `requires approval from ...` instead of `requires validation from ...`.